addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -1440,7 +1440,8 @@ var VideoWidget = MediaWidget.extend({
             embedURL = `${matches.vine[0]}/embed/simple`;
             type = 'vine';
         } else if (matches.vimeo && matches.vimeo[3].length) {
-            const vimeoAutoplay = autoplay.replace('mute', 'muted');
+            const vimeoAutoplay = autoplay.replace('mute', 'muted')
+                .replace('autoplay=1', 'autoplay=1&autopause=0');
```

Comment:
Mmh, I wonder if we should actually merge this in stable like that, you might want the autopause feature no? Would it make sense to add an extra setting like "Allow multiple autoplay at the same time"? Maybe something to discuss with PDE? What do you think?

Reply:
_With this PR, the autoplay videos are no longer autopause, the others remain autopause._

Here is my opinion. There are several scenarios:

1. If there is **only one video on the page**, whether it is autoplay or not, there is no reason to have this autopause option.
2. If there are **multiple videos** on the page but they are **all autoplay**. In this case it is imperative that they are all autopause off (which this PR does).
3. If there are **multiple videos** on the page but they are **all non autoplay**. I really wonder what kind of visitors would press play on the first video and then play on the second one and would not be happy that the first one is paused. The autopause seems to me very good too especially since these videos have sound on by default.
4. If there is a **mix of autoplay and non-autoplay** videos. Since autoplay videos are automatically muted, I personally don't mind if the autoplay video continues to run while I press the play button on another (non-autoplay) video.

In all these cases I don't see the added value of having a manual autopause option in the current state of things. On the other hand if in master we decide one day to allow to activate the sound on an autoplay video, I would reconsider things. As a visitor, when I start a video that has sound, I want all other videos that have sound to be paused. As long as autoplay videos are muted, manual autopause doesn't seem useful to me. I would even be in favor of having a public widget with our own autopause for videos coming from other platforms than vimeo (there is no native autopause on youtube and daylimotion).

While we're at it, an interesting feature we could implement in master for videos would be to play the video only when the visitor has scrolled enough to see the video and stop it when it leaves the screen (a bit like animation on appearance). Like here for example: https://www.tesla.com/solarpanels.

To summarize, with the options currently available I don't see a use case where the autopause option managed manually by the user would be useful. But maybe there is a use case I don't see :thinking: 

CC @pierre-pde

Reply:
> we decide one day to allow to activate the sound on an autoplay video

We can't without big hacks anyway, browser restriction.

> [...] autoplay

Sure, my worries here was not the autopause (especially since it is only a vimeo thing) but stable policy, but thank you for the summary :+1: In the end, this only impacts newly added videos anyway so no need to worry. Let's go with it.

> [...] improvements

I'll let @pierre-pde think about it if he wants to :)

Reply:
@qsm-odoo , considering this:
"While we're at it, an interesting feature we could implement in master for videos would be to play the video only when the visitor has scrolled enough to see the video and stop it when it leaves the screen "
Don't we have this kind of lazy loading already?

Reply:
I supposed not if @Guillaume-gdi suggested it as an improvement... :) I double-checked now and no we don't have that :) And even if we had lazy loading for videos, it would not automatically pause while scrolling the video out of the screen :wink: GDI cited tesla but we actually have that on our website odoo.com page too, as a custo :wink:

Reply:
Thanks! I'll create a task